buildworker.rst

```diff
@@ -111,7 +111,64 @@ machine reboots:
 
 For Linux:
 
-    * Add the following line to ``/etc/crontab``::
+    * For systemd based distributions, you can create a systemd unit file in order
+      to manage the service. Create the unit file named ``buildbot-worker.service``
+      under ``/etc/systemd/system`` and change the paths according to where the
+      buildbot-worker binary resides. You can verify its location by running
+      ``which buildbot-worker``. If you installed the buildbot-worker through
+      your package manager it would be::
+
+         [Unit]
+         Description=Buildbot Worker
+         Wants=network.target
+         After=network.target
+
+         [Service]
+         Type=forking
+         PIDFile=/home/buildbot/buildarea/twistd.pid
+         WorkingDirectory=/home/buildbot/buildarea
+         ExecStart=/usr/bin/buildbot-worker start
+         ExecReload=/usr/bin/buildbot-worker restart
+         ExecStop=/usr/bin/buildbot-worker stop
+         Restart=always
+         User=buildbot
+
+         [Install]
+         WantedBy=multi-user.target
+
+      If you installed the buildbot-worker through pip, the systemd unit
+      file will look like this::
+
+        [Unit]
+        Description=Buildbot Worker
+        Wants=network.target
+        After=network.target
+
+        [Service]
+        Type=forking
+        PIDFile=/home/buildbot/buildarea/twistd.pid
+        WorkingDirectory=/home/buildbot/buildarea
+        ExecStart=/usr/local/bin/buildbot-worker start
+        ExecReload=/usr/local/bin/buildbot-worker restart
+        ExecStop=/usr/local/bin/buildbot-worker stop
+        Restart=always
+        User=buildbot
+
+        [Install]
+        WantedBy=multi-user.target
+
+      Then run ``systemctl enable --now buildbot-worker.service`` to start the service
+      and have it activated on every reboot.
+
+      Note that using a systemd unit file, might produce some selinux warnings on systems
+      where the enforcing mode is enabled, usually related to the twistd.pid file.
+      If the service fails to start, you should check the output of
+      ``systemctl status buildbot-worker.service`` as well as the
+      ``/var/log/audit/audit.log`` file for potential issues and remedies.
+
+      |
+
+    * Alternatively you can create a cronjob. Add the following line to ``/etc/crontab``::
```

Comment:
systemd timer would also work

Reply:
I haven't dealt or used before systemd timers. By doing a quick check, are you proposing the basically add an example of the cron job line to a systemd unit as a timer?

Reply:
Please ignore my comment. A systemd timer requires a systemd service definition any way.